main.go

```diff
@@ -25,15 +28,44 @@ import (
 )
 
 func main() {
-	err := builder.APIServer.
-		// +kubebuilder:scaffold:resource-register
-		WithResourceAndHandler(&appcatv1.AppCat{}, appcat.New()).
-		WithResourceAndHandler(&appcatv1.VSHNPostgresBackup{}, postgres.New()).
-		WithoutEtcd().
+
+	var appcatEnabled bool = true
+	var vshnEnabled bool = false
+	var err error
+
+	appcatEnabled, err = strconv.ParseBool(os.Getenv("APPCAT_HANDLER_ENABLED"))
+	if err != nil {
+		klog.Fatal("Can't parse APPCAT_HANDLER_ENABLED env variable")
+	}
+	vshnEnabled, err = strconv.ParseBool(os.Getenv("VSHN_POSTGRES_BACKUP_HANDLER_ENABLED"))
```

Comment:
Nitpick: the variable name could be a bit confusing, as it doesn't contain `backup` and enables the backuphandler.